org.eclipse.winery.repository.rest/src/test/java/org/eclipse/winery/repository/rest/resources/entitytypeimplementations/nodetypeimplementations/NodeTypeImplementationResourceTest.java

```diff
@@ -17,7 +17,27 @@
 import org.junit.Test;
 
 public class NodeTypeImplementationResourceTest extends AbstractResourceTest {
+	private final String ENTITY_TYPE = "nodetypeimplementations/";
+	private final String INSTANCE_XML_PATH = "entityimplementations/" + ENTITY_TYPE + "instance.xml";
```

Comment:
Can this be generalized? It could be implemented in an (abstract) super class and use the use method above.

Alternatively, the paths should not be encoded in constants, but directly put at the test cases, which makes them more self-contained.